src/scripts/seqbox_cmd.py

```diff
@@ -88,38 +88,30 @@ def add_cultures(args):
                   f"culture.")
 
 
-def is_extraction_info_present(extraction_info):
-    if (extraction_info['date_extracted'] == '') & (extraction_info['extraction_identifier'] == ''):
-        return False
-    elif (extraction_info['date_extracted'] != '') & (extraction_info['extraction_identifier'] != ''):
-        return True
-    else:
-        print(f"extraction_date and extraction_identifier must both be present or both be absent for this line "
-              f"{extraction_info}. Exiting.")
-        sys.exit(1)
-
 
 def add_extractions(args):
     all_extractions_info = read_in_as_dict(args.extractions_inhandle)
     for extraction_info in all_extractions_info:
         # need to check this because, as the user will be submitting a unified sample sheet, there may be samples
         # that dont have extractions associated with them.
-        if is_extraction_info_present(extraction_info) is True:
+        if check_extraction_fields(extraction_info): 
```

Comment:
I prefer to be explicit about checking what the return from check_extraction_fields is, i.e. use `if check_extraction_fields(extraction_info) is True`

Reply:
change this everywhere in the code please.